### PR TITLE
Add Vs install script

### PR DESCRIPTION
**Pull request by jkotalik**
For https://github.com/aspnet/AspNetCore-Internal/issues/1505.

Didn't bother parsing the korebuild.json file for required components (as they will be removed soon).

### CODE REVIEW

File: scripts/install_vs.ps1 [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+$intermedateDir = "$PSScriptRoot\obj"
+mkdir $intermedateDir -ErrorAction Ignore | Out-Null
+
+$bootstrapper = "$intermedateDir\vs_enterprise1.exe"
+Invoke-WebRequest -Uri 'https://aka.ms/vs/15/pre/vs_enterprise.exe' -OutFile $bootstrapper
+
+$vsJson = "$PSScriptRoot\VsRequirements\vs.json"
+# no backslashes - this breaks the installer
+$vsInstallPath = "${env:ProgramFiles(x86)}\Microsoft Visual Studio\aspnetci\Enterprise"
```
Comment (natemcmaster) [inline scripts/install_vs.ps1]: This should probably use the default VS 2017 path.

File: scripts/install_vs.ps1 [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+)
+
+$ErrorActionPreference = 'Stop'
+Set-StrictMode -Version 1
+
+$intermedateDir = "$PSScriptRoot\obj"
+mkdir $intermedateDir -ErrorAction Ignore | Out-Null
+
+$bootstrapper = "$intermedateDir\vs_enterprise1.exe"
+Invoke-WebRequest -Uri 'https://aka.ms/vs/15/pre/vs_enterprise.exe' -OutFile $bootstrapper
```
Comment (natemcmaster) [inline scripts/install_vs.ps1]:
```suggestion
Invoke-WebRequest -Uri 'https://aka.ms/vs/15/release/vs_enterprise.exe' -OutFile $bootstrapper
```

File: scripts/install_vs.ps1 [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        Write-Error "Operation was blocked - the computer does not meet the requirements"
+    }
+    elseif (($process.ExitCode -eq 5004) -or ($process.ExitCode -eq 1602)) {
+        Write-Error "Operation was canceled"
+    }
+    elseif ($process.ExitCode -ne 0) {
+        Write-Error "Installation failed on $(hostname) for unknown reason"
+    }
+}
+finally {
```
Comment (natemcmaster) [inline scripts/install_vs.ps1]: You can probably remove the whole try/finally block. It doesn't appear to add anything.

**Review by natemcmaster — COMMENTED** [review]
Just a couple tweaks.

Also, can you add mention of this to docs/BuildFromSource.md?

File: scripts/install_vs.ps1 (hunk 1 above)
Comment (jkotalik) [reply scripts/install_vs.ps1]: whoops!

**Review by natemcmaster — APPROVED** [review]

**Pull request merged**